add_tasks.js

```diff
@@ -0,0 +1,46 @@
+var arrayTasks = [];
+
+function addTask(e) {
+
+    if (e.keyCode === 13) {
+        arrayTasks.unshift(document.getElementsByClassName("task_add_new")[0].value);
+
+        showTask();
+        document.getElementsByClassName("task_add_new")[0].value = "";
+    }
+
+}
+
+function showTask() {
+
+    var parentElem = document.body.getElementsByClassName("tasks_all")[0];
```

Comment:
bad naming, it's not just any parent, it's task container

Reply:
changed